.github/workflows/ingest-test-fixtures-update-pr.yml

```diff
@@ -37,9 +37,6 @@ jobs:
           make install-ci
 
   setup_ingest:
-    strategy:
-      matrix:
-        python-version: [ "3.8","3.9","3.10","3.11" ]
```

Comment:
this would put everything on 3.10. there are still [a number of ingest tests ](https://github.com/Unstructured-IO/unstructured/blob/main/test_unstructured_ingest/test-ingest.sh#L49-L58)we DO run through the entire matrix

Reply:
But what would that mean for the update CI job? A different PR for each version? Since they run all tests if the python version if `3.10`, I thought this made sense.

Reply:
update CI job only runs in 3.10. the fixtures themselves hopefully don't change based on python version, but we do want to test select flows across all versions to make sure the code/flow doesn't break across versions

Reply:
Isn't that what the CI job is for? This should only be to generate new outputs if they've changed.

Reply:
...well..that's how it did run, until it was updated recently

Reply:
ohh sorry, I thought this was CI.yaml, my bad
yes, let's DEFINITELY undo this change..

Reply:
...I never realized that the matrix was introduced to the ingest script which is what I think confused me

Reply:
Sorry, still confused, what reason should the update ingest test job be running through the matrix? Only `3.10` runs through all the ingest tests anyways.

Reply:
Or do you mean lets DEFINITELY get rid of the matrix?

Reply:
yes, get rid of it in the ingest test fixture workflow..so return it to how it was

Reply:
...which is what you're doing here